src/kubectl.ts

```diff
@@ -272,3 +279,8 @@ function path(context: Context): string {
     const bin = baseKubectlPath(context);
     return binutil.execPath(context.shell, bin);
 }
+
+function getKubectlArgs(context: Context): string {
+    const skipTlsVerification: boolean = context.host.getConfiguration('vs-kubernetes')[`vs-kubernetes.skip-tls-verification`];
+    return `--insecure-skip-tls-verify=${skipTlsVerification}`;
```

Comment:
I'm wondering if rather than making this a one-off we should support custom flags, so that users could also inject certificates.  I guess the problem with that is that if we later add integrated support that could emit a particular flag then we'd have a problem where it could conflict with a custom flag.